gallery/src/components/NxModal/NxModalFormExample.tsx

```diff
@@ -70,6 +72,9 @@ export default function NxModalFormExample() {
                 <span>NxModal header with form content</span>
               </h2>
             </header>
+            <NxTile.HeaderSubtitle>
```

Comment:
Needs to be adjusted along with the other modal example

Reply:
> Needs to be adjusted along with the other modal example

Updated here: 10fb738